src/Components/FavoriteCard/FavoriteCard.js

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { Link } from "react-router-dom";
+import "./FavoriteCard.css";
+
+const FavoriteCard = ({ id, name, city, state, type, isFavorite, favorites, setFavorites }) => {
+
+  const removeFavorite = (e, id) => {
+    e.preventDefault();
+    isFavorite = false;
+    const updatedFavorites = favorites.filter(favorite => favorite.id !== id);
+    setFavorites(updatedFavorites);
+  };
+
+  return (
+    <Link to={`/${id}`} style={{textDecoration: "none"}}>
+      <div className="favorite-card-wrapper">
+        <button className="remove-favorite-button" onClick={(e) => removeFavorite(e, id)}>Remove</button>
```

Comment:
Giving the user the ability to also delete a brewery from the list is nice additional feature that I should probably also implement